test/images/clusterapi-tester/Makefile

```diff
@@ -12,20 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: push
+BIN=clusterapi-tester
+ARCH ?= amd64
+TARGET ?= $(CURDIR)
+GOLANG_VERSION ?= latest
```

Comment:
if somebody wants to build binary directly using `make bin` this will be used. While calling from image folder this will overridden anyways!